### DISCUSSION TITLE
Can someone please explain how to access gradio from windows through WSL/Ubuntu?

### DISCUSSION THREAD

**Discussion by vdruts** [oobabooga/text-generation-webui#1155] [General]
I'm running WSL/Ubuntu and trying to access the UI in windows. I've done the port forwarding commands Ooba suggested. I have no firewall filters on for it, I have disabled anti-virus. No go... No worky. Can't find address. Gradio public link works but it's hella slow.

**Comment by PestToast** [comment]
What does it say in the terminal?
